tests/protocol/test_wire.py

```diff
@@ -442,6 +443,29 @@ def test_report_vm_status(self, *args):
         }
         self.assertEqual(json.dumps(v1_vm_status), actual.to_json())
 
+    @patch("azurelinuxagent.common.utils.restutil.http_request")
+    def test_send_event(self, mock_http_request, *args):
+        mock_http_request.return_value = MockResponse("", 200)
+
+        event_str = u'ćevapčići'
+        client = WireProtocol(wireserver_url).client
+        client.send_event("foo", event_str)
+
+        first_call = mock_http_request.call_args_list[0]
+        args, kwargs = first_call
+        method, url, data_received = args
```

Comment:
minor: i would rename data_received to body_received (request_body?) or add a comment to make it clearer that we are checking the body of the request